esphome/components/remote_base/__init__.py

```diff
@@ -576,6 +577,7 @@ async def magiquest_action(var, config, args):
     {
         cv.Required(CONF_ADDRESS): cv.hex_uint16_t,
         cv.Required(CONF_COMMAND): cv.hex_uint16_t,
+        cv.Optional(CONF_COMMAND_REPEATS, default=1): cv.hex_uint16_t,
```

Comment:
No need for hex here
```suggestion
        cv.Optional(CONF_COMMAND_REPEATS, default=1): cv.uint16_t,
```